Transfer.go

```diff
@@ -78,6 +79,14 @@ func SendPingTx(param SendPingTxParam) (string, PingResultError) {
 		return "", PingResultError(fmt.Sprintf("failed to get latest blockhash, err: %v", err))
 	}
 
+	// Add randomness to memo to ensure transaction is unique
+	randomBytes := make([]byte, 4)
+	_, err = rand.Read(randomBytes)
+	if err != nil {
+		return "", PingResultError(fmt.Sprintf("failed to generate random bytes, err: %v", err))
+	}
+	memoText := append([]byte("ping"), randomBytes...)
```

Comment:
i think we need to convert the entropy to text. memo text must be valid utf8.

Reply:
Someone has discovered I don't actually know Go at all

Reply:
i made that comment based on a lot of undisclosed assumptions